se_ompl/include/se/path_planner_ompl.hpp

```diff
@@ -437,15 +446,16 @@ template<typename FieldType>
 void PathPlannerOmpl<FieldType>::setSpaceBoundaries_m() {
   ob::RealVectorBounds bounds(3);
   const float dim = octree_ptr_->voxelDim();
-  const float buffer_m = 0.5;
+  const float buffer_m = 0.;
+
   lower_bound_ = lower_bound_v_.cast<float>() * dim;
   upper_bound_ = upper_bound_v_.cast<float>() * dim;
   bounds.setLow(0, lower_bound_v_.x() * dim - buffer_m);
   bounds.setLow(1, lower_bound_v_.y() * dim - buffer_m);
-  bounds.setLow(2, planning_params_.height_min - 0.2);
+  bounds.setLow(2,  ground_height_);
   bounds.setHigh(0, upper_bound_v_.x() * dim + buffer_m);
   bounds.setHigh(1, upper_bound_v_.y() * dim + buffer_m);
-  bounds.setHigh(2, planning_params_.height_max + 0.2);
+  bounds.setHigh(2, ground_height_ +2.8f);
```

Comment:
What does ground_height_ represent? Why the 2.8m offset? Wouldn't it be simpler to have an environment height and set the z limits to 0 and environment_height_ (e.g. 3 m)?

Reply:
you don't want the drone to crash into the ceiling

Reply:
adding new param ceiling_height